### ISSUE TITLE
3.5.0 > Clicking on React Date Input Field Doesn't Open Calendar Modal

### ISSUE THREAD

**Issue by TheWanderingWalnut** [cypress-io/cypress#5479]
### Current behavior:

We are using the AirBnB React Datepicker component (https://github.com/airbnb/react-dates) in our webapp and are seeing a bug in Cypress 3.5.0 which has forced us to roll back to Cypress 3.4.1, as ~18% of our tests are now broken by this change.

When clicking on the datepicker input field, we get a cursor to input text despite the field not allowing text input normally, and the calendar widget is not opened as it should be.

Screenshot from a video output by Cypress showing the failure. At this point in the test it should not have a cursor in the input field and the calendar modal should be open.

<img width="671" alt="Screen Shot 2019-10-25 at 2 47 18 PM" src="https://user-images.githubusercontent.com/56701219/67606534-84cf8d00-f736-11e9-9d8e-12a67812fe3e.png">

### Desired behavior:

When the date-picker field is clicked, the calendar widget is opened to select a day.

Below is a screenshot of the same test running in Cypress 3.4.1, which works as expected:

<img width="661" alt="Screen Shot 2019-10-25 at 2 49 26 PM" src="https://user-images.githubusercontent.com/56701219/67606592-b6485880-f736-11e9-993a-0024cd20fd08.png">

### Steps to reproduce: (app code and test code)

Click on the airbnb react datepicker with Cypress and see that it does NOT open the calendar modal, as it would when you click it manually.

### Versions

Cypress v3.5.0 (v3.4.1 works fine, we have had to roll back)
Mac OSX 10.14.6
Electron 61 / Chrome 78

**Comment by dvic** [comment]
I think this is related to #5476, I've included a reproducible test

**Comment by TheWanderingWalnut** [comment]
Great, thanks dvic, I was wondering but figured I'd log it separately for now and let the Cypress team merge them if they see fit.

**Comment by leimd** [comment]
Could this be possiblily what's causing #5459  as well?

**Comment by jennifer-shehane** [comment]
@TheWanderingWalnut 

Unfortunately we have to close this issue as there is not enough information to reproduce the problem. This does not mean that your issue is not happening - it just means that we do not have a path to move forward. 

Please comment in this issue with the exact test code to reproduce this error exactly.  and we will reopen the issue. 🙏

**Comment by TheWanderingWalnut** [comment]
@jennifer-shehane I have worked for a couple days with our developers and we have determined exactly what's breaking Cypress in our instance. We are rendering the calendar popout inside a react portal, which appears to be where Cypress trips up. I can prove that this worked in 3.4.1, and is now broken in 3.5.0 using the below repo:

https://github.com/TheWanderingWalnut/cypress-datepicker-bug

As this is breaking a substantial amount of our tests I would appreciate someone taking a look as soon as possible.

Thank You!

**Comment by jennifer-shehane** [comment]
@TheWanderingWalnut Thanks so much for providing a reproducible example.

I can confirm that this is a bug introduce in 3.5.0. Fortunately, this has already been fixed in our 3.5.1 prerelease - I ran the code against the `develop` branch and this error no longer occurs. It's very likely this was fixed as part of https://github.com/cypress-io/cypress/pull/5498

I will be closing this issue since the code for this is done. This has yet to be released. We'll update this issue and reference the [changelog](https://on.cypress.io/changelog) when it's released.
